config.json

```diff
@@ -995,7 +995,10 @@
       "core": false,
       "difficulty": 9,
       "slug": "forth",
-      "topics": null,
+      "topics": [
+        "parsing",
+        "stacks"
```

Comment:
Great suggestions! I would add `exception_handling` since the tests expect an exception to be thrown. And possibly also `strings`, `lists` and `logic`. What do you think? :)

Reply:
Completely agree, great suggestions.